.github/workflows/android-publish.yml

```diff
@@ -104,13 +120,13 @@ jobs:
           RELEASE_KEYSTORE_ALIAS: ${{ inputs.RELEASE_KEYSTORE_ALIAS }}
           RELEASE_KEYSTORE_PASSWORD: "${{secrets.RELEASE_KEYSTORE_PASSWORD}}"
           CREDENTIAL_REGISTRY_EDIT: ${{ inputs.ALLOW_ENV_EDIT }}
-          SLACK_URL: "${{ secrets.SLACK_URL }}"
+          SLACK_WEBHOOK_URL: "${{ secrets.SLACK_WEBHOOK_URL }}"
           BUILD_DESCRIPTION: ${{ inputs.BUILD_DESCRIPTION }}
 
       - name: Create Beta build and upload to play console
         if: ${{ inputs.BETA_RELEASE == 'true' }}
         run: |
-          cd android/scripts
+          cd ${{ inputs.ANDROID_SERVICE_LOCATION }}/scripts
```

Comment:
Please remove /scripts as well..

As the ANDRIOD service location will contain the complete location